doc/explanation/microcloud.rst

```diff
@@ -26,8 +26,30 @@ Networking
 
 By default, MicroCloud uses MicroOVN for networking, which is a minimal wrapper around OVN (Open Virtual Network).
 
-MicroOVN requires an uplink network that is an actual L2 subnet (which is usually not the case in a virtual cloud environment).
-In addition, MicroOVN requires its own dedicated network interface, for example, a dedicated physical network interface, a VLAN, or a virtual function on an :abbr:`SR-IOV (Single root I/O virtualisation)`-capable network interface.
+For outside connectivity, MicroOVN requires an uplink network.
+This uplink network must be an actual L2 subnet (which is usually not the case in a virtual cloud environment).
```

Comment:
Does this look better?

```suggestion
This uplink network must support broadcast and multicast traffic (so that IP adverts and packets from the OVN virtual router are transmitted and understood).
Proper Ethernet networks generally fulfil these requirements, but virtual cloud environments often don't.
```

Reply:
Yep sounds good thanks

Reply:
Maybe change "from" to "flow between the OVN virtual router and the uplink network"